code/utils/io/FileReader.py

```diff
@@ -0,0 +1,30 @@
+import logging as LOG
+from customexceptions.ReadabilityErrors import IOError
+
+
+class FileReader:
+    def __init__(self):
+        """
+        File Reader init
+        """
+        LOG.debug(f'{__name__} init')
+
+    def read(self, filename: str) -> str:
+        """
+        Read the contents of a file
+        :param filename: The name of the file to read
+        :return: The file contents if read is successful. Else None
+        """
+        LOG.debug(f'Reading file {filename}')
+        try:
+            with open(filename, 'r', encoding='utf-8') as reader:
+                tempString = ""
+                for line in reader:
```

Comment:
Also, instead of this, why not use `reader.read()` instead of reading line by line? 
For reading line by line, there seems to be a `reader.readline()`

Reply:
Bad habits (many times good!). Had been a fan of steaming and yielding... Hence used to it. In this case, cannot yield anyways. Hence will change to `reader.read()`